standard/arrays.md

```diff
@@ -120,10 +124,13 @@ Because of array covariance, assignments to elements of reference type arrays in
 > ```csharp
 > class Test
 > {
->     static void Fill(object[] array, int index, int count, object value) {
+>     static void Fill(object[] array, int index, int count, object value) 
+>     {
 >         for (int i = index; i < index + count; i++) array[i] = value;
 >     }
```

Comment:
I thought about using an expression bodied member here, but it looked wrong to me. What are your thoughts?

Reply:
I agree to not use one.

Reply:
Agreed - but we should add the braces and new lines.